src/app/zap-templates/zcl/data-model/chip/power-source-configuration-cluster.xml

```diff
@@ -26,6 +26,6 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="1"/>
-    <attribute side="server" code="0x0000" define="SOURCES" type="ARRAY" entryType="int8u" length="6" writable="false" optional="false">Sources</attribute>
+    <attribute side="server" code="0x0000" define="SOURCES" type="ARRAY" entryType="endpoint_no" length="6" writable="false" optional="false">Sources</attribute>
```

Comment:
This is in fact a change in the type: endpoint_no is 16-bit, not 8-bit.

But this is also an absolute must-have for this feature to ever work, so we do need to make this change.

Reply:
Ok, merging even if not fully backwards-compatible.